src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -10,21 +10,24 @@ import {
 import { Ajax } from 'src/libs/ajax'
 import { FormLabel } from 'src/libs/forms'
 import { useCancellation } from 'src/libs/react-utils'
+import { CloudPlatform } from 'src/pages/billing/models/BillingProject'
+import { ExternalLink } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/ExternalLink'
 import { CostCard } from 'src/pages/billing/SpendReport/CostCard'
 import { ErrorAlert } from 'src/pages/billing/SpendReport/ErrorAlert'
 
 
 const LazyChart = lazy(() => import('src/components/Chart'))
 const maxWorkspacesInChart = 10
 
-const OtherMessaging = ({ cost }) => {
+const OtherMessaging = ({ cost, cloudPlatform }) => {
   const msg = cost !== null ?
     `Total spend includes ${cost} in other infrastructure or query costs related to the general operations of Terra.` :
-    'Total spend includes infrastructure or query costs related to the general operations of Terra'
+    'Total spend includes infrastructure or query costs related to the general operations of Terra.'
   return div({ 'aria-live': cost !== null ? 'polite' : 'off', 'aria-atomic': true }, [
     span({ 'aria-hidden': true }, ['*']),
     '',
-    msg
+    msg,
+    cloudPlatform !== 'AZURE' ? '' : ExternalLink({ url: 'https://support.terra.bio/hc/en-us/articles/12029087819291-Overview-Costs-and-billing-in-Terra-on-Azure', text: ' See our documentation to learn more about Azure costs.' })
```

Comment:
You are going to have to darken the link color to pass color contrast.

Note that you can see accessibility warnings in the JavaScript console while running terra-ui locally (but not on the PR site).

![image](https://user-images.githubusercontent.com/484484/227343928-8b907c98-02bd-410d-8514-1101d7d27b44.png)

Reply:
I think the link color contrast issue is what inspired me to use a white background color here:

![image](https://user-images.githubusercontent.com/484484/227349722-acc0015a-f793-48cd-a702-00c3d34929e0.png)

Reply:
I put the space at the beginning of the link so that there wouldn't be a trailing space for GCP when this link isn't included. Happy to move it if you'd prefer

Reply:
Oh, it probably doesn't matter much either way. I was just thinking it was better not to have it with the link because if the focus is shown on the link, it looks weird to have the space at the beginning. So I guess I would prefer the trailing space on the non-link text.